Dockerfile-debian.template

```diff
@@ -27,16 +27,19 @@ RUN set -ex; \
     \
     apt-mark auto '.*' > /dev/null; \
     apt-mark manual $savedAptMark; \
-    ldd "$(php -r 'echo ini_get("extension_dir");')"/*.so \
+    extdir="$(php -r 'echo ini_get("extension_dir");')"; \
+    ldd "$extdir"/*.so \
         | awk '/=>/ { print $3 }' \
+        | awk '{print $1} {system("realpath " $1)}' \
         | sort -u \
         | xargs -r dpkg-query -S \
         | cut -d: -f1 \
         | sort -u \
         | xargs -rt apt-mark manual; \
     \
     apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/*; \
+    ldd "$extdir"/*.so | grep -qzv "=> not found";
```

Comment:
can we also add a grep to check that one or more known libs got installed ?

something like `grep -c -F "libname"`

Reply:
It appears i forgot to run `./update.sh` before committing hence the build failures :facepalm: 
I now added some output in case of failure + added the extra sanity check we at least have libzip.so in the ldd output